plugin/efficientNMSPlugin/efficientNMSInference.cu

```diff
@@ -314,12 +314,16 @@ __global__ void EfficientNMS(EfficientNMSParameters param, const int* topNumData
 
         for (int tile = 0; tile < numTiles; tile++)
         {
+            bool check_class = true;
```

Comment:
Two things
- Variabels should be camelCase, not snake_case.
- `check_class` is a misleading name because when `check_class == true` classes are not actually considered during the comparison. A better name would be `ignoreClass` or something similar.

Reply:
Would this be better?
```
for (int tile = 0; tile < numTiles; tile++)
        {
            // IOU
            if (boxIdx[tile] > i && // Make sure two different boxes are being tested, and that it's a higher index;
                boxIdx[tile] < numSelectedBoxes && // Make sure the box is within numSelectedBoxes;
                blockState == 1 &&                 // Signal that allows IOU checks to be performed;
                threadState[tile] == 0 &&          // Make sure this box hasn't been either dropped or kept already;
                (param.classAgnostic) ? true : threadClass[tile] == testClass && // Compare only boxes of matching classes when classAgnostic is false;
                lte_mp(threadScore[tile], testScore) && // Make sure the sorting order of scores is as expected;
                IOU<T>(param, threadBox[tile], testBox) >= param.iouThreshold) // And... IOU overlap.
            {
                // Current box overlaps with the box tested in this iteration, this box will be skipped.
                threadState[tile] = -1; // -1 => Mark this box's thread to be dropped.
            }
        }
```

Reply:
No this is less readable. Just changing the variable name to `ignoreClass` would be sufficient.

Reply:
Please review the new commit, thanks